renku/core/management/datasets.py

```diff
@@ -297,7 +298,7 @@ def _add_from_local(self, dataset, path, link, destination):
             else:
                 return [{
                     'path': path_in_repo,
-                    'url': src.as_uri(),
+                    'url': None,
```

Comment:
should we instead make this a relative path? It's only relevant for the user who creates the dataset but maybe that's better than `None`?

Reply:
If the file is copied from an absolute path (e.g. a shared storage mounted in the same filesystem) then this would still expose the path. Besides, a relative path is not (technically) a URL. I am open to change this though if these are fine with you.

Reply:
`file://../../some/path` is a valid URL. It's true that if you knew the location of the original project on disk you would then know the paths of the origin data, but we don't expose the absolute path of the project anywhere.